### PR TITLE
Rename virt-ui to forklift-ui

### PR DESCRIPTION
**Pull request by gildub**
Following Konveyor renaming of `virt` to `forklift`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# virt-ui
+# forklift-ui
 
 Migration Toolkit for Virtualization UI
 
-[![Build Status](https://travis-ci.com/konveyor/virt-ui.svg?branch=master)](https://travis-ci.com/konveyor/virt-ui)
+[![Build Status](https://travis-ci.com/konveyor/forklift-ui.svg?branch=master)](https://travis-ci.com/konveyor/forklift-ui)
```
Comment (mturley) [inline README.md:5]: Can you actually just remove this Travis badge? It won't work anymore now anyway, we can figure out how to put a GH Actions one in later

File: README.md [hunk 2]
```diff
@@ -16,14 +16,14 @@ A read-only preview with mock data is available at http://konveyor-virt-ui-previ
...
+cd forklift-ui
 yarn install
 ```
 
-Create a virtMeta.config.json file in the config directory using [`config/virtMeta.dev.example.json`](https://github.com/konveyor/virt-ui/blob/master/config/virtMeta.example.json) as a template. Set the `inventoryApi` property to the root URL of your virt-controller inventory API, and set the `clusterApi` property to the root URL of your host OpenShift cluster API. And also to be able to use VMware provider data to be analysed by Migration Analytics set the `inventoryPayloadApi` property to the root URL of your virt-controller inventory Payload API.
+Create a virtMeta.config.json file in the config directory using [`config/virtMeta.dev.example.json`](https://github.com/konveyor/forklift-ui/blob/master/config/virtMeta.example.json) as a template. Set the `inventoryApi` property to the root URL of your virt-controller inventory API, and set the `clusterApi` property to the root URL of your host OpenShift cluster API. And also to be able to use VMware provider data to be analysed by Migration Analytics set the `inventoryPayloadApi` property to the root URL of your virt-controller inventory Payload API.
```
Comment (mturley) [inline README.md:24]: virt-controller is referenced here, is that also changing to forklift-controller?

Comment (gildub) [reply README.md]: Yes indeed.

**Review by mturley — APPROVED** [review]
@fdupont-redhat i'm merging this for now, if we need to revert some of the changes in the Dockerfile let us know. I'm not sure about the quay.io path or the `/etc/` path.

**Pull request merged**